projects/ag-grid/documentation/code-examples/data-grid/basic/basic-data-grid-docs-demo.component.ts

```diff
@@ -63,11 +63,13 @@ export class SkyBasicDataGridDemoComponent {
   public gridData = SKY_AG_GRID_DEMO_DATA;
   public gridOptions: GridOptions;
   public searchText: string = '';
+  public noRowsTemplate;
 
   constructor(
     private agGridService: SkyAgGridService,
     private changeDetector: ChangeDetectorRef
   ) {
+    this.noRowsTemplate = `No results found`;
```

Comment:
```suggestion
    this.noRowsTemplate = `No results found.`;
```

Reply:
fixed in commit [56545e6 - "add periods."](https://github.com/blackbaud/skyux-ag-grid/pull/193/commits/56545e6d112a5dc513982f5560e10cdc4a119b42)